README.md

```diff
@@ -54,8 +49,8 @@ fn main() {
 }
 ```
 
-If you want to check for the program's output, you can use `print` or
-`print_exactly`:
+If you want to match the program's output _exactly_, you can use
+`prints_exactly`:
```

Comment:
Just a random comment.

When first poking around with this crate I found the semantics of `prints` a bit surprising. I expected a strict equality assertion. Then I learned that there is `prints_exactly`.

It's very likely a matter of taste and I might even change my mind (I often do), but I think I would prefer the strict assertion over the fuzzy assertion. I tried hard to think of a different naming, where the strict variant would have the shorter name, but I couldn't find something nice.